### ISSUE TITLE
Seawater Pump works even when its not in Water anymore

### ISSUE THREAD

**Issue by Lordnms** [MaFi-Games/Captain-of-Industry-issues#368]
**Describe the issue**
A clear and concise description of what the issue goes here.
I have the area around the Pump assigned to be Dumped with Trash, Dirt, etc. After the Pump has no connection to Water anymore because of dumping Dirt and Trash and so on around it it still Pumps Seawater.
**Expected behavior**
Explain what you expected to happen.
When there is no Connection to water anymore because of dumping stuf in the Area it shouldn't be able to work.
**To Reproduce**
1. Load attached save '...'
2. Do xxx '....'
3. Strange thing happens '...'

**Additional context**
 - Game version: [e.g. 0.2.1a, see main menu]
Early Access 0.4.4c
**Attachments**
If applicable, add save files (zipped) , screenshots, or log files to help explain the issue. You can find them in your Documents (`%userprofile%\Documents\Captain Of Industry\`). If your issue is severe, like crashes or rednering glitches, also unclude Unity logs from `%USERPROFILE%\AppData\LocalLow\Mafi Games\Captain of Industry\`. You might need to ZIP the attached files.
![20220617225245_1](https://user-images.githubusercontent.com/75038837/174402466-665e3efa-f80f-4ea4-b23f-41f878247feb.jpg)
[Lord3 .zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8931356/Lord3.zip)

**Comment by Lordnms** [comment]
This issue counts for the Liquid Dump also

**Comment by JamieRCHI** [comment]
I like this feature for ocean water pumps. I am pretending I am dumping the water into a large hidden cave.

For the liquid dumping things, usually that machine breaks for me when placing landfill over it. I suppose it is just the pipe parts that can be covered.

**Comment by utdrmac** [comment]
I disagree with this issue. What you've done is "dug a well". This is perfectly feasible. Did you know you can burry pipes? Yep. Dig a trench, lay pipe, then cover it with dirt/rock. This is the same concept.

**Comment by Lordnms** [comment]
Well in my mind when the water intake has no Connection to water it is impossible to get water. Just try to get water from Dirt/rock. The pipes are transporting water and not collecting it 
![20220618151036_1](https://user-images.githubusercontent.com/75038837/174439301-daced548-7605-453d-bb02-c4fb129faa04.jpg)
and this is what i mean with my second Post the dumped liquid would stuck in the Pipes or flowing accross the land

**Comment by JamieRCHI** [comment]
I am good with whatever the Developers decide.
Until then I would say there are two 'work arounds'.
1. Ignore it for now.
2. Don't dump near buildings of any type, and be aware of the issues if you do
3. Move the offending building.
